clients/vscode-hlasmplugin/src/test/syntaxes/endevor/parenthesis.listing

```diff
@@ -0,0 +1,129 @@
+                    NOWORKFILE                                                                                                      
+                      XREF(SHORT,UNREFS)                                                                                            
+                                                                                                                                    
+  No Overriding DD Names                                                                                                            
+»                                                                                                                                    
+                                                                                                               Page    3            
+  Active Usings: None                                                                                                               
+  Loc  Object Code    Addr1 Addr2  Stmt   Source Statement                                  HLASM R1.0  2000/01/01 00.00            
+                                      1          INSTR OP1,OP2              REMARK(REM)                                             
+                                      2          INSTR OP1,12,OP2-ARR(R15)  REMARK(REM)                                             
+                                      4          INSTR ARR(12),=SYM'F12345678'                                                      
+                                      5          INSTR OP1,=ARR(1234-2)                                                             
+                                      6          INSTR OP1,=ARR(1234-2)(something                              X                    
+                                                       something) REMARK                                                            
+                                      7          INSTR SYMBOL+12(34),=C'&&&&'                                                       
+                                     10          AIF   (&@VAR1 EQ 1).seq                                                            
+                                     11          AIF   ('&VAR' EQ 'N').seq                                                          
+                                     12          AIF   ('&VAR1' EQ 'YES    '(1,K'&VAR2)).seq                                        
+                                     13          AIF   ('&VAR'(1,1) EQ '&&').seq                                1234567             
+                                     14          AIF   ( NOT ( (('A' LE '&VAR') AND ('&VAR' LE 'B'))  OR       X                    
+                                        (('C' LE '&VAR') AND ('&VAR' LE 'D'))  REMARK                          X1234567             
+                                                                                                                                    
```

Comment:
was the input to the file not a data set?

Reply:
Do you ask because of the empty line? I'll verify what's going on. 

The input to `ASMA90` was provided in `SYSIN` from data that were read from a file

Reply:
Also because of the return code 0. This file should have failed the assembly process.

Reply:
I see. I'm manually overriding the rc to 0 so the listing generation process can proceed further. I'll recheck if it can proceed even with non-zero rc

Reply:
Also, I'm suppressing a lot of warning and error messages